docker/entrypoint.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/sh
 
+# Default to UID and GID 1000 if not provided
+USER_ID=${UID:-1000}
+GROUP_ID=${GID:-1000}
+
+# Create group and user with the specified IDs
+addgroup -g "$GROUP_ID" usergroup
+adduser -u "$USER_ID" -G usergroup username
+
+# Change ownership of the working directory
+chown -R "$USER_ID":"$GROUP_ID" "$MEDIA_PATH"
+
+
```

Comment:
At this point the container is running as root. One of the reasons to drop root is for more security.

Perhaps we can `su` the `exec` on line 23?